help/Makefile.am

```diff
@@ -1,16 +1,16 @@
-include $(top_srcdir)/gnome-doc-utils.make
-dist-hook: doc-dist-hook
+@YELP_HELP_RULES@
 
-DOC_ID = hamster
-DOC_INCLUDES = legal.xml
-DOC_LINGUAS = cs da de el es fa fr gl hu pl pt_BR ro ru sl te zh_CN zh_HK zh_TW
+HELP_ID = hamster
 
-DOC_PAGES = \
+HELP_FILES = \
         backup.page \
         index.page \
         input.page \
         keys.page \
         merge.page \
         reports.page \
         statistics.page \
-        tracking.page
+        tracking.page \
+        legal.xml
```

Comment:
This seems to add an additional `legal.xml`, which seems unrelated to the xml2po -> itstool change? If so, it might be better in a separate commit.

Reply:
Ah, I guess it was previously automatically added somehow through the `DOC_INCLUDES` variable?

Reply:
Yes, it's just a compatibility with yelp-tools, related to other variable renames.

Reply:
Hm, looking at the old [`xml2po` Makefile include](https://gitlab.gnome.org/GNOME/ekiga/blob/master/gnome-doc-utils.make), it indeed seems that `DOC_INCLUDES` was pretty much included in the translation just like the help pages, so this change makes sense. However, it seems that the waf-based processing of translations did *not* actually look at `DOC_INCLUDES`, so quite possibly this file was not processed with `xml2po` before and this fix fixed by this commit.
*however*, I tried running this just now, and it seems it's still not processed by itstool somehow. I haven't figured out why yet...

Reply:
Ah found it, it only processes *.pages. Added a comment in the relevant place and resolving this comment.